mcgill-en.csl

```diff
@@ -37,6 +37,18 @@
         <single>para</single>
         <multiple>paras</multiple>
       </term>
+      <term name="sub verbo" form="short">
+			  <single></single>
```

Comment:
Okay, great. Just checking, although I'm pretty sure Frank's extended list is already quite comprehensive, since he studied many of the available legal style guides.